posthog/api/decide.py

```diff
@@ -359,6 +359,15 @@ def _session_recording_config_response(request: HttpRequest, team: Team, token:
                 else:
                     linked_flag = linked_flag_key
 
+            rrweb_script_config = None
+            if (settings.SESSION_REPLAY_RRWEB_SCRIPT is not None) and (
+                settings.SESSION_REPLAY_RRWEB_SCRIPT_ALLOWED_TEAMS == ["*"]
+                or str(team.id) in settings.SESSION_REPLAY_RRWEB_SCRIPT_ALLOWED_TEAMS
+            ):
+                rrweb_script_config = {
+                    "script": settings.SESSION_REPLAY_RRWEB_SCRIPT,
+                }
```

Comment:
slightly over careful in returing a structure rather than a value in case this changes over time e.g. loading from a different host